### PR TITLE
Major content update on homepage

### PR DESCRIPTION
**Pull request by juice500ml**
## Why?
- Current homepage is too unfriendly to beginners

## What?
- [x] Basic structuring of the homepage
- [x] Citation cleanup + add
- [x] Update ESPnet2 tutorials
- [x] Write inference tutorial OR link to existing notebook
- [x] Write ESPnet-EZ tutorial OR link to existing notebook
- [x] Build new ESPnet-EZ folder in espnet/notebook and populate
- [x] Reformat Demos page (avoid automatic generation, leverage new markdown file within espnet/notebook repo)
- [x] Better design to show various recipes
- [x] Incorporate recipe readmes to docs
- [ ] Fill up missing recipe readmes
- [x] Clean up existing recipe readmes

## See also
- https://github.com/espnet/espnet/pull/5885
- https://github.com/espnet/espnet/pull/5847

### CODE REVIEW

**Comment by Masao-Someki** [comment]
- I removed dark mode
image

- Removed /recipe/TEMPLATE_local
image

- Removed recipe image, add icon, and changed background image for tutorials
image

**Comment by pyf98** [comment]
Thanks for the work!
The current changes are related to documentation, readme, and homepage. I do not have strong opinions on those parts.

Since many files are modified, I want to check others' opinions.  @sw005320 Sorry for bothering you during a busy period.

**Comment by Masao-Someki** [comment]
@pyf98 
I've build and uploaded this project's resources to the draft repository. Please refer to the following URL to review the PR:

https://masao-someki.github.io/espnet_draft_home_page/

**Comment by pyf98** [comment]
Thanks @Masao-Someki! It generally looks good to me. Some sentences on the homepage can be refined by some grammar-checking toolkits (or GPT?)

**Comment by juice500ml** [comment]
@pyf98 the last two commit applies your idea of using GPT to rephrase the homepage! Thanks again 👍

File: doc/index.md [hunk 1]
```diff
@@ -4,194 +4,149 @@ icon: /assets/image/espnet.png
...
-        details: "EURO: ESPnet Unsupervised Recognition - Open-source"
-        link: https://github.com/espnet/espnet?tab=readme-ov-file#uasr-unsupervised-asr-euro-espnet-unsupervised-recognition---open-source
+      - title: "Speech Codec"
+        link: ./recipe/codec1.md
+        icon: material-symbols:hd-outline
 
-      - title: "S2T: Speech-to-text with Whisper-style multilingual   multitask models"
-        link: https://github.com/espnet/espnet?tab=readme-ov-file#s2t-speech-to-text-with-whisper-style-multilingual-multitask-models
+      - title: "... And much more!"
+        link: recipe/
```
Comment (ftshijt) [inline doc/index.md:79]: Are they reserved for next PR? I saw a few tutorials are ready but not linked here

File: egs2/TEMPLATE/asvspoof1/README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Speaker Verification Spoofing and Countermeasures
```
Comment (ftshijt) [inline egs2/TEMPLATE/asvspoof1/README.md:1]: Note that the asvspoof is for espnet tutorial (new task). We might want to state that cause it is designed to be incomplete for users.

**Review by ftshijt — COMMENTED** [review]
Many thanks for the update! A few minor comments

File: doc/index.md [hunk 3]
```diff
@@ -4,194 +4,149 @@ icon: /assets/image/espnet.png
...
-  - width: 80%
-heroText: "ESPnet: end-to-end speech processing toolkit"
-tagline: "ESPnet is an end-to-end speech processing toolkit covering many speech-related tasks."
-actions:
-  - text: Get Started
-    icon: book
-    link: ./espnet2_tutorial.md
-    type: primary
+  - height: auto
+  - width: 400px
```
[the remark this answers is not in the document]
Comment (juice500ml) [reply doc/index.md:8]: image
Thanks to vuepress, it's automatically handled (to be at most 100% size of the surrounding div)!

File: doc/index.md (hunk 1 above)
Comment (juice500ml) [reply doc/index.md:79]: Good point, I also added other recipes too on 2d8ea77! Thanks for pointing out!

File: doc/citations.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
+# Additional citations
```
[the remark this answers is not in the document]
Comment (juice500ml) [reply doc/citations.md:1]: Thanks for the citations! Added on 2d8ea77

File: egs2/TEMPLATE/asvspoof1/README.md (hunk 2 above)
Comment (juice500ml) [reply egs2/TEMPLATE/asvspoof1/README.md:1]: Didn't know!! Added the relevant links at 2d8ea77! Thanks!!

**Review by ftshijt — APPROVED** [review]
LGTM! Thanks for your contribution~

**Pull request merged**